examples/docker-compose-stdout-example/docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: '2.0'
```

Comment:
Not sure why should be removed `version` field? It is not supported by new compose version?

I tested locally with restored field and tests seems to pass

Reply:
Sure I can add the version property back!

Reply:
The support for `version` property remains in compose v2. The version here defines the version of the docker compose "Schema" (here 2.0) which is quite old now. Compose-file versions evolved from v1.x, to v2.x, to v3.x, and now stabilized in compose specification. Docker recommends using compose spec when crafting compose.yaml files. In order to instruct engine to apply spec compose.yaml file should not contain version field any more.
See https://docs.docker.com/compose/compose-file/compose-versioning/ for more details on compose versioning.

This change however has nothing to do with upgrade to docker compose v2. Both v1 and v2 support all composefile versions up to the spec. I would recommend dropping version field to make use of schema validation from spec. This should be done in a separate MR to keep the scope of this MR more focused.